### PR TITLE
[PT] Responses

### PR DESCRIPTION
**Pull request by nikito7**

### CODE REVIEW

File: responses/pt/HassTurnOff.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ language: pt
...
   intents:
     HassTurnOff:
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.area }}
-      fans_area: Turned off fans in {{ slots.area }}
-      cover: Closed {{ slots.name }}
-      cover_area: Closed {{ slots.area }}
+      default: (Desligado | Desligada) (o | a) {{ slots.name }}
+      lights_area: Desligada a luz (na | no) {{ slots.area }}
+      fans_area: Desliga a ventilação (na | no) {{ slots.area }}
```
Comment (marcocunha) [inline responses/pt/HassTurnOff.yaml:7]: Should be `Desligada`

**Review by marcocunha — CHANGES_REQUESTED** [review]

**Comment by schizza** [comment]
And you should stick to `no whitespaces around |` as in #830

**Review by marcocunha — APPROVED** [review]

**Review by marcocunha — APPROVED** [review]

File: responses/pt/HassLightSet.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ language: pt
...
   intents:
     HassLightSet:
-      brightness: 'Brilho (do | da) {{ slots.name }} definido para {{ slots.brightness }}'
-      brightness_area: Brilho (no | na) {{ slots.area }} definido para {{ slots.brightness }}
-      color: 'Cor (do | da) {{ slots.name }} definida para {{ slots.color }}'
-      color_area: Cor (na | no) {{ slots.area }} defenida para {{ slots.color }}
+      brightness: 'Brilho (do|da) {{ slots.name }} definido para {{ slots.brightness }}'
+      brightness_area: Brilho (no|na) {{ slots.area }} definido para {{ slots.brightness }}
+      color: 'Cor (do|da) {{ slots.name }} definida para {{ slots.color }}'
+      color_area: Cor (na|no) {{ slots.area }} defenida para {{ slots.color }}
```
Comment (marcocunha) [inline responses/pt/HassLightSet.yaml]: typo `definida`

**Review by marcocunha — CHANGES_REQUESTED** [review]

File: responses/pt/HassLightSet.yaml (hunk 2 above)
Comment (nikito7) [reply responses/pt/HassLightSet.yaml]: Old commit

**Pull request closed without merging**